frontend/src/pages/Common/Budget.js

```diff
@@ -320,10 +320,16 @@ const Budget = (props) => {
             variant="contained"
             color="secondary"
```

Comment:
Orange color is ugly. I know you didn't author this, but please consider changing to color="primary"

Reply:
I changed the color of the button since there was no reason for secondary color.